psi4/src/psi4/detci/ints.cc

```diff
@@ -724,19 +740,15 @@ void CIWavefunction::read_dpd_ci_ints() {
 void CIWavefunction::rotate_dfmcscf_twoel_ints(SharedMatrix Uact,
                                                SharedVector twoel_out) {
     // => Rotate twoel ints <= //
-    int nQ = dferi_->size_Q();
+    int nQ = dfh_->get_naux();
     int nrot = CalcInfo_->num_rot_orbs;
     int nact = CalcInfo_->num_ci_orbs;
     int nav = nact + CalcInfo_->num_rsv_orbs;
 
     // Read RaQ
-    std::shared_ptr<Tensor> RaQT = dferi_->ints()["RaQ"];
     SharedMatrix RaQ(new Matrix("RaQ", nrot, nact * nQ));
-
     double* RaQp = RaQ->pointer()[0];
-    FILE* RaQF = RaQT->file_pointer();
-    fseek(RaQF, 0L, SEEK_SET);
-    fread(RaQp, sizeof(double), nrot * nact * nQ, RaQF);
+    dfh_->fill_tensor("RaQ", RaQ);
```

Comment:
Yes, all forms of `fill_tensor`, `write_disk_tensor`, etc. greatly simply I/O.  The recursive signatures work wonderfully IMO.